Casamari/MNC019.xml

```diff
@@ -0,0 +1,796 @@
+                        </decoNote>
+                        <decoNote xml:id="b2" type="bindingMaterial">
+                           <material key="wood"/>  
+                           <material key="leather"/>    
+                           <material key="textile"/>   
+                        </decoNote>
+                        <decoNote xml:id="b3" type="Cover" color="dark brown">Full cover. The leather is decorated with blind tooled ornaments: double lines, 
+                           double circles, criss-cross, and others.</decoNote>
+                        <decoNote xml:id="b4" type="SewingStations">4</decoNote>
+                        <decoNote xml:id="b5" type="Spine">Two pairs of chains of the endband stitches are visibile at the top and the bottom of the spine cover.</decoNote>
+                        <decoNote xml:id="b6" type="Other">
+                           Threads of various origin are inserted in the outer margins for navigating the text. 
+                           More specifically, red <term key="leafTabMark">leaf tab markers</term> are inserted in the upper outer margin of <locus target="#19 #36 #74 #115 #133"/>
+                           and uncoloured <term key="leafStringMark">leaf string markers</term> are inserted in the upper outer margin of <locus target="#91 #109"/>.
+                        </decoNote>
+                     </binding>
+                  </bindingDesc>
+               </physDesc>
+          
+               <history>
+                  <origin>
+                     <origDate notBefore="1800" notAfter="1899" evidence="lettering">
+                        The handwriting is consistent with a 19th century dating.
+                     </origDate>
+                     <note/>
+                  </origin>
+               </history>
+          
+               <additional>
+                  <adminInfo>
+                     <recordHist>
+                        <source>
+                           <listBibl type="secondary">
+                              <bibl>
+                                 <ptr target="bm:Britaetc2017ThreeCollections"/>
+                                 <citedRange>184</citedRange>
+                              </bibl>
+                              </listBibl>
+                        </source>
+                     </recordHist>
+                  </adminInfo>
+               </additional>
+            </msDesc>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <projectDesc>
+            <p>Encoded according to TEI P5 Guidelines.</p>
+         </projectDesc>
+         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+            <xi:fallback>
+               <p>Definitions of prefixes used.</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc>
+         <particDesc>
+            <listPerson>
```

Comment:
I think I never provide this information. Is this important for the catalogue? Shall I also do it?

Reply:
In reality it is mostly redundant. An owner/scribe/donor/patron is nearly always indicated as such in a colophon, an additional note or a supplication formula. So, while entering or commenting those portions of text we can add `role="donor"` (or whatever, scribe, owner etc) inside persName (e.g. `<persName ref="PRS13014TaklaHa" role="scribe">ተክለ፡ ሐዋርያት።</persName>` from MNC014). In this way, all the relevant names can be extracted by the system and listed somewhere in the printed catalogue (e.g. in a prosopography section). The other cases, in which we know that a certain individual was involved in the history of the ms but (s)he is not mentioned in the Ms, can be anyway encoded under `<history><origin/></history>` or `<history><provenance/></history>`.
So, I frankly find `<listPerson/>` basically useless.